client/src/components/GeocoderInput.vue

```diff
@@ -40,6 +44,10 @@ export default defineComponent({
   mounted() {
     this.geocoder.addTo('.geocoder');
     this.geocoder.setPlaceholder(this.placeholder);
+    
+    if (this.acceptedTypes != null) {
+      this.geocoder.setTypes(this.acceptedTypes.join());
```

Comment:
This looks weird without a comma in the `join` call, since I assume `setTypes` takes a comma-separated list. But if it works then please ignore.

Reply:
It does, and join does by default but this is a good point, I'll add it just for clarity and so if join behavior changes this won't break.